include/umf/memory_pool_ops.h

```diff
@@ -41,13 +41,72 @@ struct umf_memory_pool_ops_t {
     /// \param pool pool to finalize
     void (*finalize)(void *pool);
 
-    /// Refer to memory_pool.h for description of those functions
+    /// \brief Allocates \p size bytes of uninitialized storage of \p pool
```

Comment:
Could you please also align comments with the corresponding functions from the `memory_pool.h` file